src/templates/ira_doe_workflow_high_efficiency_gas_furnace.mdx

```diff
@@ -134,8 +135,8 @@
       Photo of the electronic methane leak detector while it is testing a gas connection
     </Photo>
 
-    <Photo id="inlet_gas_pressure_monometer_photo" label="Monometer Showing Inlet Gas pressure – Photo" required>
-      Photo of the manometer measuring the inlet gas pressure
+    <Photo id="outlet_gas_pressure_monometer_photo" label="Monometer Showing Outlet Gas pressure – Photo" required>
```

Comment:
There is a minor difference in the ID for this Photo component compared to PhotoInput: 'outlet_gas_pressure_monometer_photo'. Replace the 'o' with 'a' in 'manometer' to get 'outlet_gas_pressure_manometer_photo'.

Reply:
I changed "monometer" to "manometer" in both the id and the label.